### PR TITLE
Code optimization && Propagande 

### PR DESCRIPTION
**Pull request by ghost**

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -70,232 +63,224 @@ module.exports.ProfileHooker = async (settings = { username: "", cookie: "", tim
...
+            if (!node.edge_sidecar_to_children)
+              posts.push({ name: `instaHooker_${node.shortcode}_0_${node.id}.png`, url: node.display_url });
+
+
+        }
+      })
+  }
 }
 
+/**
```
Comment (ghost) [inline index.js:257]: Remove jsdoc comment since this is just a cli app, no API included.

**Comment by ghost** [comment]
Thanks for the PR, everything looks good but did you checked it? Is it running normally with your changes? If you send proof that it's working, i'll accept the PR.

File: index.js [hunk 2]
```diff
@@ -70,232 +63,224 @@ module.exports.ProfileHooker = async (settings = { username: "", cookie: "", tim
...
-        timeout: settings.timeout ? settings.timeout : 50,
-        output: settings.output || "instahooker-out"
-    }
-
-    const posts = [];
+  const options = {
+        "url": settings.url,
+        "cookie": `sessionid=${settings.cookie};`,
+        "timeout": settings.timeout ? settings.timeout : 50,
+        "output": settings.output || "instahooker-out"
```
Comment (ghost) [inline index.js:70]: Remove double quotes from object keys, they're unnecessary.

File: index.js [hunk 3]
```diff
@@ -70,232 +63,224 @@ module.exports.ProfileHooker = async (settings = { username: "", cookie: "", tim
...
-        cookie: `sessionid=${settings.cookie};`,
-        timeout: settings.timeout ? settings.timeout : 50,
-        output: settings.output || "instahookerStory-out",
-        highlight: settings.highlight || false
+  const options = {
+    "username": settings.username,
+    "cookie": `sessionid=${settings.cookie};`,
+    "timeout": settings.timeout ? settings.timeout : 50,
+    "output": settings.output || "instahookerStory-out",
+    "highlight": settings.highlight || false
```
Comment (ghost) [inline index.js:95]: Remove double quotes from here too.

File: package.json [hunk 4]
```diff
@@ -1,16 +1,16 @@
 {
-  "name": "new",
-  "version": "1.0.0",
-  "description": "Fetching posts from instagram.",
-  "main": "index.js",
   "author": "shynox",
-  "license": "MIT",
   "dependencies": {
     "commander": "^8.3.0",
     "fs": "^0.0.1-security",
```
Comment (ghost) [inline package.json:5]: Remove fs from here. It's built-in with nodejs.

File: package.json [hunk 5]
```diff
@@ -1,16 +1,16 @@
...
   },
+  "description": "Fetching posts from instagram.",
   "devDependencies": {
     "@types/node-fetch": "^3.0.3"
-  }
+  },
+  "license": "MIT",
+  "main": "index.js",
+  "name": "new",
+  "version": "1.0.0"
```
Comment (ghost) [inline package.json:15]: Move these to the top except `devDependencies`.

Comment (ghost) [reply package.json:15]: Increase the version by minor. New version must be `1.1.0.`

**Review by ghost — CHANGES_REQUESTED** [review]

**Comment by wioniqle-q** [comment]
> Thanks for the PR, everything looks good but did you checked it? Is it running normally with your changes? If you send proof that it's working, i'll accept the PR.

is an untested comit

**Comment by wioniqle-q** [comment]
> Thanks for the PR, everything looks good but did you checked it? Is it running normally with your changes? If you send proof that it's working, i'll accept the PR.

jsdoc is a matter of choice I did for 2015 es and there is no point in removing it

File: package.json (hunk 4 above)
Comment (wioniqle-q) [reply package.json:5]: > 

edited to match the lesser character

File: index.js (hunk 3 above)
Comment (wioniqle-q) [reply index.js:95]: > 

they have been changed to as name space there is no vulnerability in the removal

File: index.js (hunk 2 above)
Comment (wioniqle-q) [reply index.js:70]: > 

they have been changed to namespace I don't need to remove

**Comment by ghost** [comment]
Please check the reviews and change requests. After you finishing reviews and commit, i'll merge.

**Comment by wioniqle-q** [comment]
> Please check the reviews and change requests. After you finishing reviews and commit, i'll merge.

I don't have time to do this. If you want to put it together, you can fix it

**Comment by ghost** [comment]
> > Please check the reviews and change requests. After you finishing reviews and commit, i'll merge.
> 
> I don't have time to do this. If you want to put it together, you can fix it

Then I'm closing the PR. You created a PR and you're not maintaining it and telling something like `do it yourself`. If you really want to these changes merged, just commit the reviews. But we don't need anymore. Thanks for your effort, though.

**Comment by wioniqle-q** [comment]
> 
if wtb, do it yourself

**Pull request closed without merging**